### ISSUE TITLE
LuauRegenerateSourcemap error

### ISSUE THREAD

**Issue by realtica** [lopi-py/luau-lsp.nvim#13]
Hello, autocomplete is not working and LuauRegenerateSourcemap gives me this error:

```
Error executing Lua callback: ...hare/nvim/plugged/luau-lsp.nvim/lua/luau-lsp/command.lua:19: attempt to index field 'uv'
 (a nil value)                                                                                                           
stack traceback:                                                                                                         
        ...hare/nvim/plugged/luau-lsp.nvim/lua/luau-lsp/command.lua:19: in function <...hare/nvim/plugged/luau-lsp.nvim/l
ua/luau-lsp/command.lua:18>   
```

This is my config:

```
vim.o.exrc = true
require("luau-lsp").setup {
  fflags = {
    sync = true, -- sync currently enabled fflags with roblox's published fflags
    override = {
      LuauTarjanChildLimit = 0,
    },
  },
  server = {
    settings = {
      -- https://github.com/folke/neoconf.nvim/blob/main/schemas/luau_lsp.json
      ["luau-lsp"] = {
        completion = {
          imports = {
            enabled = true, -- enable auto imports
          },
        },
      },
    },
  },
  sourcemap = {
    enabled = true,
    autogenerate = true, -- automatic generation when the server is attached
    rojo_project_file = "default.project.json"
  },
  types = {
    roblox = true,
    roblox_security_level = "PluginSecurity",
  },
}
```

**Comment by realtica** [comment]
Thanks, now I received this error:
![image](https://github.com/lopi-py/luau-lsp.nvim/assets/852446/c4954520-f7fc-4173-baea-9e472b222abf)

**Comment by lopi-py** [comment]
Try the latest commit, I fixed that (you can also provide an argument to the command like `:LuauRegenerateSourcemap path/to/your.project.json`, but yeah, it should also work even without passing an argument which is what I just fixed), but actually it should run automatically as you have `sourcemap.autogenerate` enabled, can you check your logs with `:LuauLog`? Also make sure you have the server running with `:LspInfo`

**Comment by lopi-py** [comment]
`Invalid project file provided` should be fixed with 4146be6, feel free to reopen if this is not the case
